docs/for_contributors/building_documentation.md

```diff
@@ -7,7 +7,7 @@ Libtropic documentation is built using the two following frameworks, each buildi
 Normally, you should not need to build the documentation yourself - it is available on our [GitHub Pages](https://tropicsquare.github.io/libtropic/latest/), where versions for the [master](https://github.com/tropicsquare/libtropic/tree/master) branch and all [releases](https://github.com/tropicsquare/libtropic/releases) are automatically built and released by our GitHub Actions. However, in the case of contributing to the documentation, it is handy to be able to build it locally and preview the new changes. Refer to the following sections for steps on how to do that.
 
 ## Installation of the Dependencies
-!!! important
```

Comment:
Wouldn't it be better to incorporate Python dependency to the text as step 1?  e.g,

1. Make sure you have Python 3 installed, at least version 3.8.

The mkdocs doc suggest that recent version is required and mkdocs material requires at least 3.8. So we should mention this as well instead of 3.x.